Demo/SampleViewController.swift

```diff
@@ -10,6 +10,7 @@ import UIKit
 import SwiftyPickerPopover
 
 class SampleViewController: UIViewController, UICollectionViewDataSource, UICollectionViewDelegate {
+    var selecttedValue: Int = 0
```

Comment:
If I must say, it should be named selectedRow.

Reply:
```
// for keeping selectedRow
private var selectedRow: Int = 0
```

Reply:
ok :)